test/integration/Android/FiltersTests.cs

```diff
@@ -0,0 +1,68 @@
+//Licensed under the Apache License, Version 2.0 (the "License");
+//you may not use this file except in compliance with the License.
+//See the NOTICE file distributed with this work for additional
+//information regarding copyright ownership.
+//You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS,
+//WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//See the License for the specific language governing permissions and
+//limitations under the License.
+
+using System.Collections.Generic;
+using NUnit.Framework;
+using OpenQA.Selenium.Appium.Android;
+using OpenQA.Selenium.Appium;
+
+namespace Appium.Net.Integration.Tests.helpers
+{
+    [TestFixture]
+    public class FiltersTests
+    {
+        private AndroidDriver _driver;
+
+        [OneTimeSetUp]
+        public void BeforeAll()
+        {
+            var capabilities = Env.ServerIsRemote()
+                ? Caps.GetAndroidUIAutomatorCaps(Apps.Get("androidApiDemos"))
```

Comment:
these two branches are the same, aren't they?

Reply:
Which branches do you mean?

Reply:
I mean the part after `?` and after `:`

Reply:
Yeah, they are the same, I just took it from another test we have. wasn't paying extra attention.
Fixed